doc/rules-docc.md

```diff
@@ -0,0 +1,51 @@
+<!-- Generated with Stardoc: http://skydoc.bazel.build -->
+
+Defines rules for building Apple DocC targets.
+
+<a id="docc_archive"></a>
+
+## docc_archive
+
+<pre>
+docc_archive(<a href="#docc_archive-name">name</a>, <a href="#docc_archive-default_code_listing_language">default_code_listing_language</a>, <a href="#docc_archive-dep">dep</a>, <a href="#docc_archive-diagnostic_level">diagnostic_level</a>, <a href="#docc_archive-enable_inherited_docs">enable_inherited_docs</a>,
+             <a href="#docc_archive-fallback_bundle_identifier">fallback_bundle_identifier</a>, <a href="#docc_archive-fallback_bundle_version">fallback_bundle_version</a>, <a href="#docc_archive-fallback_display_name">fallback_display_name</a>, <a href="#docc_archive-kinds">kinds</a>,
+             <a href="#docc_archive-transform_for_static_hosting">transform_for_static_hosting</a>)
+</pre>
+
+
+Builds a .doccarchive for the given dependency.
+The target created by this rule can also be `run` to preview the generated documentation in Xcode.
+
+NOTE: At this time Swift is the only supported language for this rule.
```

Comment:
Thanks!

Reply:
Let's update the issue or create a new issue for Objective-C support once this has been merged.